onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -72,20 +72,11 @@ ORT_API(void, OrtApis::ReleaseKernelInfo, _Frees_ptr_opt_ OrtKernelInfo*) {
 namespace onnxruntime {
 namespace standalone {
 
-void ReleaseNode(onnxruntime::Node* node) {
-  if (node) {
-    for (auto* input_arg : node->InputDefs()) {
-      delete input_arg;
-    }
-    for (auto* output_arg : node->OutputDefs()) {
-      delete output_arg;
-    }
-    delete node;
-  }
-}
-
-using NodePtr = std::unique_ptr<onnxruntime::Node, void (*)(onnxruntime::Node*)>;
+using NodePtr = std::unique_ptr<onnxruntime::Node>;
 using StandAloneNodesMap = InlinedHashMap<const onnxruntime::OpKernel*, NodePtr>;
+using ArgPtr = std::unique_ptr<onnxruntime::NodeArg>;
+using ArgPtrs = absl::InlinedVector<ArgPtr, 10>;
```

Comment:
why is 10 used as the inlined size instead of the default?

Reply:
let's go with default.